docs/asciidoc/static/upgrading.asciidoc

```diff
@@ -31,3 +31,43 @@ This procedure downloads the relevant Logstash binaries directly from Elastic.
 4. Test your configuration file with the `logstash --configtest -f <configuration-file>` command. Configuration options for 
 some Logstash plugins have changed in the 2.0 release.
 5. Restart your Logstash pipeline after updating your configuration file.
+
+=== Upgrading Logstash and Elasticsearch to 2.0
+
+If you are using Elasticsearch as an output, and wish to upgrade to Elasticsearch 2.0, please be
+aware of https://www.elastic.co/guide/en/elasticsearch/reference/2.0/breaking-changes-2.0.html[breaking changes]
+before you upgrade. In addition, the following steps needs to be performed after upgrading to Elasticsearch 2.0:
+
+1. **Mapping changes:** Logstash software upgrade will not automatically update your existing template 
+in Elasticsearch. Users may have custom template changes, so by default an upgrade will leave the 
+template as is. 
+
+Note: There is one known issue (removal of https://www.elastic.co/guide/en/elasticsearch/reference/1.4/mapping-object-type.html#_path_3[path]) with using GeoIP filter that needs a manual update to the template.
+
+If you have custom template changes, please make sure to save it and merge any changes. You can 
+get the existing template by running:
+
+```
+curl -XGET localhost:9200/_template/logstash
+```
+
+Add the following option to your Logstash config:
```

Comment:
I'd include something here to the effect of:

"Even if you don't have a custom template, Logstash will not overwrite an existing template by default.  If you are not using a custom template, you can automatically update to whatever is included with Logstash by setting `template_overwrite => true`, like this:"